Source/Advance.cpp

```diff
@@ -83,13 +87,21 @@ PeleC::do_mol_advance(
   if (verbose != 0) {
     amrex::Print() << "... Computing MOL source term at t^{n} " << std::endl;
   }
-  FillPatcherFill(Sborder, 0, NVAR, numGrow() + nGrowF, time, State_Type, 0);
+
+  int nGrow_Sborder = numGrow() + nGrowF;
```

Comment:
This is a good idea. I would just ask that you pick another name. It can be confusing that `nGrow_Sborder` is not the same as `Sborder.nGrow()`. Maybe rename to `nGrow_FP_border`?